package.json

```diff
@@ -22,76 +22,29 @@
   },
   "main": "./dist/opentype.js",
   "browser": "./dist/opentype.js",
-  "module": "./dist/opentype.module.js",
+  "module": "./dist/opentype.mjs",
   "scripts": {
     "build": "npm run b:umd && npm run b:esm",
     "dist": " npm run d:umd && npm run d:esm",
-    "test": "npm run build && npm run dist && mocha --require reify --recursive && npm run lint",
```

Comment:
I think we still need the ``--recursive``

Reply:
you are right (226 vs 317)